panels/nerf_3dview_panels/dataset_panel.py

```diff
@@ -1,9 +1,11 @@
 import bpy
+import os
```

Comment:
```suggestion
from pathlib import Path
```

Let's use pathlib.  More comments below.